resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesExecutorBackend.scala

```diff
@@ -82,7 +82,7 @@ private[spark] object KubernetesExecutorBackend extends Logging {
         clientMode = true)
 
       var driver: RpcEndpointRef = null
-      val nTries = 3
+      val nTries = sys.env.getOrElse("EXECUTOR_DRIVER_PROPS_FETCHER_MAX_ATTEMPTS", 3).toInt
```

Comment:
Do we need to document this parameter? Or it is just one internal parameter for test purpose only?

Reply:
Thank you for review. This is an internal parameter like `.internal` config. However, this code path is invoked to get `SparkConf`. So, we cannot use `SparkConf` value. We may document this env variable later in some debug section.